### ISSUE TITLE
Travis: Crawler run failing in master (related to direct mode)

### ISSUE THREAD

**Issue by kyleam** [datalad/datalad#3217]
Until a02e6da88 (TST: travis: Widen selector for extension runs, 2019-03-08), our crawler Travis run only executed a small subset of the tests.  Now that this is brought into master, a crawler test fails (below) because master has dropped direct mode.  I'm opening this issue here because the failure is happening in core, but I think the main fix would have to happen on crawler's side.

<details>
<summary>Crawler failures</summary>

https://travis-ci.org/datalad/datalad/jobs/504695887

```
======================================================================
ERROR: datalad_crawler.nodes.tests.test_annex.test_add_archive_content_tar(True,)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 1112, in newfunc
    reraise(*exc_info)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/six.py", line 693, in reraise
    raise value
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 1084, in newfunc
    ret = func(*args, **kwargs)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 431, in newfunc
    return t(*(arg + (d,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad_crawler/nodes/tests/test_annex.py", line 193, in _test_add_archive_content_tar
    largefiles="exclude=*.txt and exclude=SOMEOTHER")
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad_crawler/nodes/annex.py", line 300, in __init__
    self.repo = (GitRepo if no_annex else AnnexRepo)(path, always_commit=False, **kwargs)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/support/repo.py", line 146, in __call__
    instance = type.__call__(cls, *new_args, **new_kwargs)
TypeError: __init__() got an unexpected keyword argument 'direct'
======================================================================
ERROR: datalad_crawler.nodes.tests.test_annex.test_add_archive_content_tar(False,)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/nose/case.py", line 198, in runTest
    self.test(*self.arg)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 1112, in newfunc
    reraise(*exc_info)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/six.py", line 693, in reraise
    raise value
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 1084, in newfunc
    ret = func(*args, **kwargs)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/tests/utils.py", line 431, in newfunc
    return t(*(arg + (d,)), **kw)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad_crawler/nodes/tests/test_annex.py", line 193, in _test_add_archive_content_tar
    largefiles="exclude=*.txt and exclude=SOMEOTHER")
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad_crawler/nodes/annex.py", line 300, in __init__
    self.repo = (GitRepo if no_annex else AnnexRepo)(path, always_commit=False, **kwargs)
  File "/home/travis/virtualenv/python3.5.6/lib/python3.5/site-packages/datalad/support/repo.py", line 146, in __call__
    instance = type.__call__(cls, *new_args, **new_kwargs)
TypeError: __init__() got an unexpected keyword argument 'direct'

```

</details>

**Comment by yarikoptic** [comment]
I believe those were addressed in https://github.com/datalad/datalad-crawler/pull/36. I will just wait for https://github.com/datalad/datalad-crawler/pull/29 to get merged, and then kick out release

**Comment by kyleam** [comment]
> I believe those were addressed in [datalad/datalad-crawler#36](https://github.com/datalad/datalad-crawler/pull/36). I will just wait for [datalad/datalad-crawler#29](https://github.com/datalad/datalad-crawler/pull/29) to get merged, and then kick out release

Wonderful.  Thanks!

**Comment by yarikoptic** [comment]
issues were addressed with 0.4 release of the crawler - our master is green again!
